src/window.c

```diff
@@ -1526,27 +1526,6 @@ column 0.  */)
 				  0, false);
 }
 
-DEFUN ("window-point", Fwindow_point, Swindow_point, 0, 1, 0,
-       doc: /* Return current value of point in WINDOW.
-WINDOW must be a live window and defaults to the selected one.
-
-For a nonselected window, this is the value point would have if that
-window were selected.
-
-Note that, when WINDOW is selected, the value returned is the same as
-that returned by `point' for WINDOW's buffer.  It would be more strictly
-correct to return the top-level value of `point', outside of any
-`save-excursion' forms.  But that is hard to define.  */)
-  (Lisp_Object window)
-{
-  register struct window *w = decode_live_window (window);
-
-  if (w == XWINDOW (selected_window))
-    return make_number (BUF_PT (XBUFFER (w->contents)));
```

Comment:
You don't seem to be covering this case in the Rust implementation -- it's always getting the marker position. Am I missing something?

Reply:
I forgot it, but I still don't understand why this has to be handled separately. But I fxied it. Thanks.